src/block-management/block-picker.js

```diff
@@ -4,18 +4,24 @@
  */
 
 /**
- * WordPress dependencies
- */
-import { SVG } from '@wordpress/components';
-
+* External dependencies
+*/
 import React from 'react';
+import { FlatList, Text, TouchableHighlight, View } from 'react-native';
+
+/**
+* WordPress dependencies
+*/
+import { SVG } from '@wordpress/components';
+import { BottomSheet } from '@wordpress/editor';
 import { Component } from '@wordpress/element';
-import { FlatList, Text, TouchableHighlight, View, Dimensions } from 'react-native';
-import Modal from 'react-native-modal';
+import { getBlockTypes } from '@wordpress/blocks';
+
+/**
+* Internal dependencies
+*/
 import styles from './block-picker.scss';
 import { name as unsupportedBlockName } from '../block-types/unsupported-block';
-// Gutenberg imports
-import { getBlockTypes } from '@wordpress/blocks';
 
 type PropsType = {
 	style?: StyleSheet,
```

Comment:
looks like we don't need safeAreaBottomInset anymore

and we can also get rid of this at the bottom of the page:

BlockPicker.defaultProps = {
	safeAreaBottomInset: 0,
};